File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L5/06/looping.kt
package IR

import java.util.Scanner

fun main() {
    println("Miles\t\tKilometers")
    println("-------------------------------")
    var mil = 1
    print(mil)
}

fun print(mil: Int) {
    for (i in mil..10) {
        println("$i\t\t${i * 1.609}")
    } // looping dari 1 sampai 10
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L5/06/looping.ts
import { Scanner } from 'util';

class Looping {
  static main(args: string[]): void {
    console.log('Miles\t\tKilometers');
    console.log('-------------------------------');
    let mil: number;
    this.print(mil = 1);
  }

  static print(mil: number): void {
    for (let i = mil; i <= 10; i++) {
      console.log(`${i}\t\t${i * 1.609}`);
    } // looping dari 1 sampai 10
  }
}

